evm/src/main/java/org/hyperledger/besu/evm/operation/BaseFeeOperation.java

```diff
@@ -34,8 +33,7 @@ public Operation.OperationResult executeFixedCostOperation(
       final MessageFrame frame, final EVM evm) {
     final Optional<Wei> maybeBaseFee = frame.getBlockValues().getBaseFee();
     if (maybeBaseFee.isEmpty()) {
-      return new Operation.OperationResult(
-          OptionalLong.of(gasCost), Optional.of(ExceptionalHaltReason.INVALID_OPERATION));
+      return new Operation.OperationResult(gasCost, ExceptionalHaltReason.INVALID_OPERATION);
     }
     frame.pushStackItem(maybeBaseFee.orElseThrow());
     return successResponse;
```

Comment:
same q about static OperationResult(2, null) here

Reply:
Basefee gets overridden in downstream clients, so it cannot be optimized.